charts/flyte-core/values.yaml

```diff
@@ -356,6 +356,16 @@ db:
 #
 
 configmap:
+  # Details of cluster
+  clusters: []
+#    - name: "cluster_1"
+#      endpoint: { { your-cluster-1-kubeapi-endpoint.com } }
+#      enabled: true
+#      auth:
+#        type: "file_path"
+#        tokenPath: "/var/run/credentials/cluster_1_token"
+#        certPath: "/var/run/credentials/cluster_1_cacert"
```

Comment:
Should we add a way to add these in `values.yaml`? or is the expectation to use `secrets:` section?

Reply:
we don't need that, User can create secret separately before control plane deployment  and use additional Volume to mount secret in flyteadmin pod.